.github/workflows/pong-server.yaml

```diff
@@ -34,10 +34,18 @@ jobs:
                   pip3 install isort
                   pip3 install -r pong_server/docker/requirements.txt
 
-            - name: Run unit tests
+            - name: Copy shared files
+              run: |
+                  cp -r pong_server/src/shared_code pong_server/src/game_creator/shared_code
+
+            - name: Run unit tests game creator
               working-directory: pong_server/
               run: |
-                  python3 -m unittest discover test
```

Comment:
Why do you remove unittest in pong_server?

Reply:
The only tests that were written for pong server were for the redirection server which is now deprecated